Source/IntegrationTests/TestFiles/LitTests/LitTest/verification/Inputs/outOfResourceAndIsolateAssertions.check

```diff
@@ -1,23 +1,23 @@
 CHECK: Verified 0/2 symbols. Waiting for f to verify.
-CHECK: Verified part #0, 1/10 of f, on line 5, verified successfully \(time: .*, resource count: .*\)
-CHECK: Verified part #1, 2/10 of f, on line 5, verified successfully \(time: .*, resource count: .*\)
-CHECK: Verified part #2, 3/10 of f, on line 5, verified successfully \(time: .*, resource count: .*\)
-CHECK: Verified part #3, 4/10 of f, on line 5, verified successfully \(time: .*, resource count: .*\)
-CHECK: Verified part #4, 5/10 of f, on line 5, verified successfully \(time: .*, resource count: .*\)
-CHECK: Verified part #5, 6/10 of f, on line 5, verified successfully \(time: .*, resource count: .*\)
-CHECK: Verified part #6, 7/10 of f, on line 5, verified successfully \(time: .*, resource count: .*\)
-CHECK: Verified part #7, 8/10 of f, on line 5, verified successfully \(time: .*, resource count: .*\)
-CHECK: Verified part #8, 9/10 of f, on line 5, verified successfully \(time: .*, resource count: .*\)
-CHECK: Verified part #9, 10/10 of f, on line 5, verified successfully \(time: .*, resource count: .*\)
+CHECK: Verified 1/10 of f: assertion at line 5, verified successfully \(time: .*, resource count: .*\)
```

Comment:
This reformatting is definitely an improvement. In the future it would be nice to be more specific about what part of the assertion is verified in this batch though.